cmd/kind/version/version.go

```diff
@@ -31,9 +31,8 @@ func NewCommand() *cobra.Command {
 		Use:   "version",
 		Short: "prints the kind CLI version",
 		Long:  "prints the kind CLI version",
-		RunE: func(cmd *cobra.Command, args []string) error {
+		Run: func(cmd *cobra.Command, args []string) {
 			println(Version)
```

Comment:
something else here.

this PR does not change the stderr -> stdout for version, because `println` remains in use.
it only changes `RunE` -> `Run`.

`println` always prints to stderr.

to achieve what you want you need to use:
`fmt.Println()`

or `fmt.Fprintf(os.Stdout....`